### ISSUE TITLE
some menus are not exposed to screen readers properly

### ISSUE THREAD

**Issue by w0jrl** [home-assistant/frontend#25127]
### Checklist

- [x] I have updated to the latest available Home Assistant version.
- [x] I have cleared the cache of my browser.
- [x] I have tried a different browser to see if it is related to my browser.
- [ ] I have tried reproducing the issue in [safe mode](https://www.home-assistant.io/blog/2023/11/01/release-202311/#restarting-into-safe-mode) to rule out problems with unsupported custom resources.

### Describe the issue you are experiencing

When creating automations in the companion app on Android, I am unable to do things like test actions and delete actions with the TalkBack screen reader.  The menu that has those actions works fine on Linux using Firefox with the Orca screen reader. I am able to expand or collapse existing actions, and even create new actions. However, if I make a mistake, I have to redo the entire automation from scratch. I'm not able to delete parts of it. 

### Describe the behavior you expected

The menu that has test, disabled, delete, etc. should be exposed as an accessible object.

### Steps to reproduce the issue

1. Turn on TalkBack in Android accessibility settings. 
2. Open the home assistant companion app. 
3. Add a new automation. 
4. In the new automation, under actions, add an action, then try to edit it. 
Remember, when using TalkBack, you need to double tap on the screen instead of single tap to perform actions.

### What version of Home Assistant Core has the issue?

2025.4.2

### In which browser are you experiencing the issue?

companion app

### Which operating system are you using to run this browser?

Android 15

### State of relevant entities

```txt

```

### Problem-relevant frontend configuration

```yaml

```

### Javascript errors shown in your browser console/inspector

```txt

```

**Comment by w0jrl** [comment]
I hope that the description that I gave on the menu that I'm referring to is adequate enough. I do not know what this menu looks like visually.
